twine-upload.sh

```diff
@@ -53,7 +54,37 @@ environments like GitHub Actions without needing to use username/password \
 combinations or API tokens to authenticate with PyPI. Read more: \
 https://docs.pypi.org/trusted-publishers"
 
-if [[ "${INPUT_USER}" == "__token__" && -z "${INPUT_PASSWORD}" ]] ; then
+ATTESTATIONS_WITHOUT_TP_WARNING="::warning title=attestations setting ignored::\
+The workflow was run with 'attestations: true', but an explicit password was \
```

Comment:
This seems ambiguous. It may imply a key in the `permissions` setting (because it says “workflow”) but I suppose you're talking about the action input, right?

Reply:
Yep, I'll clarify -- as-is the workflow itself doesn't require the `attestations` permission from GHA itself, but I can see how this would snare users.

Reply:
Done, PTAL 🙂 -- this input is arguably still unfortunately named, but given that it'll be flipped to default after the beta I _think_ that's an acceptable tradeoff?

Alternatively, I could rename it to something like `beta-attestations:` or `x-attestations:` or similar, to make it less ambiguous.